openapi_tester/configuration.py

```diff
@@ -20,11 +20,7 @@ def schema_loader(self):
 
     @property
     def case_tester(self) -> Callable:
-        return django_settings.OPENAPI_TESTER.get('CASE_TESTER', lambda *args: None)
-
-    @property
-    def camel_case_parser(self) -> bool:
-        return django_settings.OPENAPI_TESTER.get('CAMEL_CASE_PARSER', False)
+        return django_settings.OPENAPI_TESTER.get('CASE_TESTER', None)
```

Comment:
Won't this reintroduce this issue?

https://github.com/snok/django-openapi-tester/issues/83#issuecomment-702828307

Reply:
see my next comment.